energyplus_regressions/diffs/table_diff.py

```diff
@@ -414,6 +455,14 @@ def table_diff(thresh_dict, inputfile1, inputfile2, abs_diff_file, rel_diff_file
 
         uheading1 = uheadings1[i1]
 
+        # There are some (for now one) tables that we will want to skip entirely because they are not useful for
+        # throwing regressions, add search keys to this list to skip them
+        completely_skippable_table_keys = [
+            'Object Count Summary_Entire Facility_Input Fields'
+        ]
+        if any([x in uheading1 for x in completely_skippable_table_keys]):
+            continue
```

Comment:
First off, if a table name is listed here, it can be completely skipped.  It won't be included in the equal value count or anything.  This should only be used for very specific table names.  As of now it is only skipping the input fields table that lists how many autosizable fields and such.